src/main/java/com/synopsys/integration/alert/channel/email/EmailAddressHandler.java

```diff
@@ -120,11 +125,11 @@ public FieldAccessor updateEmailAddresses(String providerConfigName, MessageCont
         return Set.of();
     }
 
-    private Set<String> collectProviderEmailsFromProject(String providerConfigName, String projectName, boolean projectOwnerOnly) {
+    private Set<String> collectProviderEmailsFromProject(String providerConfigName, String projectHref, boolean projectOwnerOnly) {
         Optional<ProviderProject> optionalProject = providerDataAccessor.getProjectsByProviderConfigName(providerConfigName)
                                                         .stream()
-                                                        .filter(project -> project.getName().equals(projectName))
-                                                        .findFirst(); // FIXME use href
```

Comment:
I was under the impression that changing to the use of href was a much more significant change than just these few lines.